src/frontend/src/pages/FinancePage/RefundsSection.tsx

```diff
@@ -121,8 +121,9 @@ const Refunds = ({ userReimbursementRequests, allReimbursementRequests }: Refund
     return <LoadingIndicator />;
 
   const displayedReimbursements = allReimbursements && tabValue === 1 ? allReimbursements : userReimbursements;
-  const displayedReimbursementRequests =
-    allReimbursementRequests && tabValue === 1 ? allReimbursementRequests : userReimbursementRequests;
+  const displayedReimbursementRequests = (
+    allReimbursementRequests && tabValue === 1 ? allReimbursementRequests : userReimbursementRequests
+  ).filter((request: ReimbursementRequest) => request.reimbursementStatuses[0].type !== ReimbursementStatusType.DENIED);
```

Comment:
This looks at the first status, we actually want to check the most recent (last status in the array)